user-guide/Advanced_Functionality/Swarming/SwarmingElements.md

```diff
@@ -4,17 +4,150 @@ uid: SwarmingElements
 
 # Swarming elements
 
-With DataMiner Swarming, you can swarm basic elements from one DataMiner Agent in a cluster to another. At present, the *Swarming* [soft-launch option](xref:SoftLaunchOptions) must be enabled for this.
+With DataMiner Swarming, you can swarm basic elements from one DataMiner Agent in a cluster to another.
 
-When you are swarming an element so it gets hosted on a different DataMiner Agent, a temporary transition occurs.
-
-To indicate this, the following message will be displayed:
+When you are swarming an element so it gets hosted on a different DataMiner Agent, a temporary transition occurs. To indicate this, the following message will be displayed:
 
 ```txt
-'ElementName' is currently switching.
+'ElementName' is currently swarming.
            More info
 ```
 
-During this transition, the ability to open element cards or change element configuration will be temporarily suspended.
+During this transition, the ability to open element cards or change element configuration for the involved element will be temporarily suspended. Once the element migration is complete, it will become accessible again.
+
+> [!NOTE]
+> To be able to trigger swarming for an element, you need the *Swarming* user permission as well as config rights on the element.
+
+## Swarming an element from Cube
+
+From Cube, elements can be swarmed from *System Center > Agents > Status > Swarm...*
+
+## Swarming an element from Automation
```

Comment:
add reference to catalog package with such a script?
https://catalog.dataminer.services/details/ffb0166d-9394-4f14-abd0-48e2175484a0

Reply:
or github repo?
https://github.com/SkylineCommunications/SLC-AS-SwarmElements

Reply:
linked to both as a TIP